aspnetcore/grpc/health-checks.md

```diff
@@ -92,6 +91,17 @@ There are two methods on the `Health` service:
 * `Check` is a unary method for getting the current health status. Health checks are executed immediately when `Check` is called. The server returns a `NOT_FOUND` error response if the client requests an unknown service name. This can happen at app startup if health results haven't been published yet.
 * `Watch` is a streaming method that reports changes in health status over time. <xref:Microsoft.Extensions.Diagnostics.HealthChecks.IHealthCheckPublisher> is periodically executed to gather health results. The server returns an `Unknown` status if the client requests an unknown service name.
 
+The `Grpc.HealthCheck` client can be used in a client factory approach:
+
+```csharp
+builder.Services
+    .AddGrpcClient<Health.HealthClient>(o =>
+    {
+        o.Address = new Uri("https://localhost:5001");
+    });
+```
+In the previous example, a client factory for `Health.HealthClient` instances is registered with the dependency injection system. Then, these instances are injected into services for executing health check calls.
```

Comment:
It would be useful to have a link to client factory page for people who don't know what it is.

```suggestion
In the previous example, a client factory for `Health.HealthClient` instances is registered with the dependency injection system. Then, these instances are injected into services for executing health check calls.

For more information, see <xref:grpc/clientfactory>.
```